kafka-connect-neo4j/src/test/kotlin/streams/kafka/connect/common/ConfigurationMigratorTest.kt

```diff
@@ -0,0 +1,178 @@
+package streams.kafka.connect.common
+
+import com.fasterxml.jackson.core.type.TypeReference
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import streams.kafka.connect.source.SourceType
+import java.io.File
+
+
+class ConfigurationMigratorTest {
+
+  @Test
+  fun `should migrate keys to new configuration`() {
+    // Given a configuration containing normal keys
+    val originals =
+      mapOf(
+        "neo4j.topic.pattern.merge.node.properties.enabled" to "true",
+        "neo4j.server.uri" to "neo4j+s://x.x.x.x",
+        "neo4j.retry.max.attemps" to "1"
+      )
+
+    // When the configuration is migrated
+    val migratedConfig = ConfigurationMigrator(originals).migrate()
+
+    // Then the keys are updated to new key format containing the original value
```

Comment:
nit: assert the size of `migratedConfig` as well?

Reply:
done